packages/core/data-transfer/lib/engine/index.ts

```diff
@@ -1,9 +1,14 @@
+import { isEmpty } from 'lodash';
+import { uniq } from 'lodash/fp';
 import type {
+  Diff,
   IDestinationProvider,
   ISourceProvider,
   ITransferEngine,
   ITransferEngineOptions,
 } from '../../types';
+import calculateSchemaDiffs from '../strategies';
+import { jsonDiffs } from '../utils';
```

Comment:
```suggestion
```
Looks like this isn't used

Reply:
> I'm getting a failing test in `packages/core/data-transfer/lib/providers/local-strapi-source-provider/__tests__/index.test.ts` because the category field is not being included in getSchemas

Yes, that is because it's not in the selectedKeys array, I will change the test